docs/_docs/templates.md

```diff
@@ -418,6 +419,11 @@ The default is `default`. They are as follows (with what they filter):
 
 ## Tags
 
+* Includes
+* Code snippet highlighting
+* Linking to pages, collections and posts (the new and improved way)
+* Linking to posts (the old way)
+
```

Comment:
Yes, but I had changed several of the sections' headers not realizing it would break links, so I took those changes back and didn't add links to this TOC. I'm adding them now.